tracer/src/Datadog.Trace/Configuration/ImmutableIntegrationSettings.cs

```diff
@@ -51,5 +51,16 @@ internal ImmutableIntegrationSettings(string name)
         /// that determines the sampling rate for this integration.
         /// </summary>
         public double AnalyticsSampleRate { get; }
+
+        /// <summary>
+        /// Gets the <c>ImmutableIntegrationSettings</c>  as a string.
+        /// </summary>
+        /// <remarks>
+        /// To simplify viewing in the debugger.
+        /// </remarks>
+        /// <returns>
+        /// The object in a human readable format.
+        /// </returns>
+        public override string ToString() => $"{this.IntegrationName} - {(this.Enabled == true ? "Enabled" : "Disabled")} - Analytics: {(this.AnalyticsEnabled == true ? $"Sample Rate {this.AnalyticsSampleRate}" : "Disabled")}";
```

Comment:
A `[DebuggerDisplay]` attribute would be a better option if you want to see this variable in the debugger. cf [the doc](https://docs.microsoft.com/en-us/visualstudio/debugger/using-the-debuggerdisplay-attribute?view=vs-2022)